packages/events/src/event.rs

```diff
@@ -11,6 +12,8 @@ use tendermint::{abci, block};
 use crate::errors::DecodingError;
 use crate::Error;
 
+pub type AbciEventTypeFilter = String;
```

Comment:
```suggestion
pub struct AbciEventTypeFilter {
  pub event_type: String,
};
```
so that we can extend it later.